src/libraries/System.Net.Http.Json/tests/UnitTests/TranscodingReadStreamTests.cs

```diff
@@ -232,15 +232,6 @@ public Task ReadAsync_Works_WhenInputIs_Unicode(string message)
             return ReadAsyncTest(sourceEncoding, message);
         }
 
-        [Theory]
-        [MemberData(nameof(ReadAsyncInputLatin), "utf-7")]
-        [MemberData(nameof(ReadAsyncInputUnicode), "utf-7")]
-        public Task ReadAsync_Works_WhenInputIs_UTF7(string message)
-        {
-            Encoding sourceEncoding = Encoding.UTF7;
```

Comment:
To be honest, I debated this. In the end I asked myself two questions:

> Does the library intend to provide special support for UTF-7? Does this test provide additional coverage beyond what is already covered by the existing tests for UTF-16, UTF-32, ASCII, and Latin-1?

Since I believe the answer to these questions is no I felt the test could be safely deleted. But if there's a counterargument to be made this is definitely the right forum! :)

Reply:
IMO the `Transcoding*` classes are most intended for UTF-16 -> UTF-8 and vice versa and also as you mention, we are no losing coverage so I think is fine to delete the tests. Thanks for explaining.